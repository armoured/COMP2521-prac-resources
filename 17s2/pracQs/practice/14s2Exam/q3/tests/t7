./q3 < tests/t7.in
