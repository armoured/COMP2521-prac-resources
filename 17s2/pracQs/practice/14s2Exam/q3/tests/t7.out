Graph:
# vertices = 12, # edges = 12

0 is connected to 1 2
1 is connected to 0
2 is connected to 0
3 is connected to 4 5 6 7
4 is connected to 3
5 is connected to 3
6 is connected to 3
7 is connected to 3
8 is connected to 9 10 11
9 is connected to 8 10 11
10 is connected to 8 9 11
11 is connected to 8 9 10

Well-connected vertices
3 has 4 connections
8 has 3 connections
9 has 3 connections
10 has 3 connections
11 has 3 connections
0 has 2 connections
